### PR TITLE
Feature/cax 1 prs@159 Added scripts to register a connector resource and to consume a resource.

### PR DESCRIPTION
**Pull request by ouphi**
Added scripts to register a connector resource and a consumer resource.
idsapi.py and resourceapi.py scripts have been taken in the Dataspace connector repo in [the scripts/test folder](https://github.com/International-Data-Spaces-Association/DataspaceConnector/tree/main/scripts/tests).
I would like to avoid to change them.
The intention of this PR is to have a clear README to make it easy to create and consume a resource.

### CODE REVIEW

File: coreservices/partsrelationshipservice/connectors_scripts/README.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+```bash
+pipenv sync
+pipenv shell
+./create_catalog_and_artifact.py \
+<provider-url> \
+<provider-internal-alias> \
+<artifact-title> \
+<access-url-to-access-the-artifact> \
+<admin> \
```
Comment (marcgs) [inline coreservices/partsrelationshipservice/connectors_scripts/README.md:16]: username?

File: coreservices/partsrelationshipservice/connectors_scripts/resourceapi.py [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+    def create_artifact(self, data={"value": "SOME LONG VALUE"}):
+        response = self.session.post(self.recipient + "/api/artifacts", json=data)
+        return response.headers["Location"]
+
+    def create_contract(
+        self,
+        data={
+            "start": "2021-04-06T13:33:44.995+02:00",
+            "end": "2021-12-06T13:33:44.995+02:00",
```
Comment (marcgs) [inline coreservices/partsrelationshipservice/connectors_scripts/resourceapi.py:52]: this is going to expire soon, is this relevant?

File: coreservices/partsrelationshipservice/connectors_scripts/consume_artifact.py [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+# Negotiate contract
+obj = offer["ids:contractOffer"][0]["ids:permission"][0]
+obj["ids:target"] = provider_artifact_url
+response = consumer.contractRequest(
+    provider_alias + "/api/ids/data", offer_url, provider_artifact_url, False, obj
+)
+pprint.pprint(response)
+
+# Pull data
```
Comment (marcgs) [inline coreservices/partsrelationshipservice/connectors_scripts/consume_artifact.py:87]: I think there are 2 part of this script: the contract negotiation and the data access. Contract negotiation should only be done once AFAIU, after this there is an agreement that can be used multiple times to access data. Perhaps we should think to split this script into these 2 parts (no need to do this in this PR though)

Comment (ouphi) [reply coreservices/partsrelationshipservice/connectors_scripts/consume_artifact.py:87]: I assume that they will save the `/data` url to access the API afterwards no?

File: coreservices/partsrelationshipservice/connectors_scripts/resourceapi.py (hunk 2 above)
Comment (ouphi) [reply coreservices/partsrelationshipservice/connectors_scripts/resourceapi.py]: Hmm good catch. I can increase it.

Comment (ouphi) [reply coreservices/partsrelationshipservice/connectors_scripts/resourceapi.py]: I changed the code to make sure the contract starts yesterday and end in a year.

File: coreservices/partsrelationshipservice/connectors_scripts/consume_artifact.py (hunk 3 above)
Comment (marcgs) [reply coreservices/partsrelationshipservice/connectors_scripts/consume_artifact.py]: Ok we can assume that. Perhaps it is worth documenting this.

**Review by marcgs — APPROVED** [review]

File: coreservices/partsrelationshipservice/connectors_scripts/consume_artifact.py (hunk 3 above)
Comment (ouphi) [reply coreservices/partsrelationshipservice/connectors_scripts/consume_artifact.py]: Good point, I documented this.

File: coreservices/partsrelationshipservice/connectors_scripts/README.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
+# Interact with Connectors
+
+Our PRS API should be accessible via a consumer connector. An artifact linked to our API is created via a provider connector.
```
Comment (cpeeyush) [inline coreservices/partsrelationshipservice/connectors_scripts/README.md:3]:
```suggestion
PRS API should be accessible via a consumer connector. An artifact linked to our API is created via a provider connector.
```

**Review by cpeeyush — APPROVED** [review]

File: coreservices/partsrelationshipservice/connectors_scripts/README.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
+# Interact with Connectors
+
+PRS API should be accessible via a consumer connector. An artifact linked to our API is created via a provider connector.
```
Comment (cpeeyush) [inline coreservices/partsrelationshipservice/connectors_scripts/README.md:3]:
```suggestion
PRS API should be accessible via a consumer connector. An artifact linked to PRS API is created via a provider connector.
```

**Pull request merged**